front/types/assistant/agent.ts

```diff
@@ -59,7 +59,8 @@ export type AgentGenerationConfigurationType = {
 export type GlobalAgentStatus =
   | "active"
   | "disabled_by_admin"
-  | "disabled_missing_datasource";
+  | "disabled_missing_datasource"
+  | "require_upgrade";
```

Comment:
NIT: `disabled_free_workspace` to follow the convention disable + reason

Reply:
yes let's update :+1: